src/indexer/allo/v1/handlers/handleProjectCreatedEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { EventHandlerArgs } from "chainsauce";
+import { parseAddress } from "../../../../address.js";
+import { fullProjectId } from "../fullProjectId.js";
+import type { Indexer } from "../../../indexer.js";
+import type { Changeset } from "../../../../database/index.js";
+
+export async function handleProjectCreatedEvent(
+  args: EventHandlerArgs<
+    Indexer,
+    "AlloV1/ProjectRegistry/V1" | "AlloV1/ProjectRegistry/V2",
```

Comment:
So this would handle indexing both v1 and v2 ?

Reply:
I noticed there are 2 abis referencing the "ProjectCreated" event:

- src/indexer/abis/allo-v1/v1/ProjectRegistry.ts
- src/indexer/abis/allo-v1/v2/ProjectRegistry.ts

Both belonging to allo-v1

"ProjectCreated" event is only handled in src/indexer/allo/v1/handleEvent.ts 

Therefore this PR is just moving the logic to a separate file, not changing anything else.

I am not aware if currently there are events on both cases: `"AlloV1/ProjectRegistry/V1" | "AlloV1/ProjectRegistry/V2"` 

But in order to get the right type of EventHandlerArgs, it has to be <Indexer, {AbiNames}, {EventName}>